v2/manifests/core-ons/dp-feedback-api.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  dp-feedback-api:
+    x-repo-url: "https://github.com/ONSdigital/dp-feedback-api"
+    build:
+      context: ${DP_REPO_DIR:-../../../..}/dp-feedback-api
+      dockerfile: Dockerfile.local
```

Comment:
dp-feedback-api has no Dockerfile.local

Reply:
Added with this [PR](https://github.com/ONSdigital/dp-feedback-api/pull/16)